### PR TITLE
Create DICO_milestone_1.md

### PR DESCRIPTION
**Pull request by DICO-TEAM**
add DICO milestone 1

# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/636

### CODE REVIEW

**Comment by Noc2** [comment]
Thanks for the delivery. We will look into it as soon as possible.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks again for the delivery. I started to look into your delivery. The last delivery of the milestone seems to be missing in your PR: “Lending module(research oriented)”. Could you add it or comment on it? Also the front-end integration doesn’t seem to be open source. Is this correct? In general, when I compile your code, I get a lot of warnings. It would be nice if you could improve this. Furthermore, it seems like you haven’t published a demo video or where can I find this?

**Comment by DICO-TEAM** [comment]
@Noc2 thanks for your work

1.Lending module (research oriented): When we initially thought about integrating lending into the project, but because too many teams in Polkadot are doing work similar to defi, so we put our work focus on DAO and ICO. 
`milestone 1` does not say that this implementation must be included, it just for research oriented， but we have no plans to remove this module.  We will do some work  in the next stage

2.front-end : we will open source it 

3. warnings: yes, have some warnings, we should fix it.

https://github.com/DICO-TEAM/dico-chain/issues/127


4. some video about DICO:

https://www.youtube.com/watch?v=gt1BfWZaVWs
https://www.youtube.com/watch?v=we9feIzIJW8
https://www.youtube.com/watch?v=qQuMZ87G2UU

**Review by Noc2 — CHANGES_REQUESTED** [review]
Sorry for the very long delay here and thanks for the detailed reply. Regarding the lending module the specification of the delivery currently says that you “publish a blog to show the test results.” If you no longer plan on publishing this, you need to create a PR/amendment to update the original contract. Thanks for open sourcing the front-end, but it currently doesn’t contain any information on how to compile and run it as well as potential tests. Could you please add this? Furthermore, thanks for sharing the videos, but none seems to be a “demo video that will explain how users can start using the platform for KYC and ICO.” or also not all of them are in english, so it’s very hard to follow them. And let me know, once you have improved the code.

**Comment by semuelle** [comment]
Any updates here, @DICO-TEAM?

**Comment by Noc2** [comment]
I’m closing this PR due to inactivity. Let me know if I should reopen it. Also, it would be appreciated if you could give us a status update on the grant.
